### PR TITLE
Secondary button styles

### PR DESCRIPTION
**Pull request by AlexandrKolesnikov**

### CODE REVIEW

**Comment by Alexei-Kekin** [comment]
Sanya God boy

File: src/styles/index.scss [hunk 1]
```diff
@@ -31,3 +31,14 @@ body {
     background-color: rgba($color-firm7, 0.7);
   }
 }
+
+.button--secondary {
```
Comment (AlexandrKolesnikov) [inline src/styles/index.scss]: Example comment

**Comment by Haidai-Oksana** [comment]
I want pink buttons !imoptant

**Comment by ElenaRise** [comment]
God loves you

File: src/styles/index.scss (hunk 1 above)
Comment (AlexandrKolesnikov) [reply src/styles/index.scss]: Done

**Comment by TonyXdddd** [comment]
pink color so cool :D

**Pull request merged**